index.js

```diff
@@ -64,6 +70,9 @@ module.exports = function(bot){
           {
             send: function(msg){
               sendMessage(bot,message,msg);
+            },
+            messageRoom: function (msg, room) {
+              sendMessagetoRoom(bot, msg, room)
```

Comment:
It would also potentially be beneficial to return out the original bot object as well as the send function here.  That will give us flexibility in the future for jobs to interact with other hubot specific functions like pulling from the brain.